index.html

```diff
@@ -40,4 +41,143 @@ <h1 class="heading">
         </ul>
       </div>
     </section>
-    
\ No newline at end of file
+    <section class="work-section">
```

Comment:
- [ ] Good job so far 👍 your site looks good and it follows the template design but you'll notice that when leaving from the first page to the projects page, there is a change in the background color and the bottom left side of the first page is curved. It would be good if you could implement this so that your site could be more accurate.
- Your design:
![Capture1](https://user-images.githubusercontent.com/85113567/153600276-2d19fe6b-2d2c-4ab9-8e1d-5554c44f6605.PNG)
- Figma design:
![Capture2](https://user-images.githubusercontent.com/85113567/153600355-2be9f0cc-2dbd-4631-8427-bebe85c2fa0e.PNG)